admin-dev/themes/new-theme/scss/components/_timeline.scss

```diff
@@ -1,123 +1,163 @@
 .timeline {
   position: relative;
-  display: table;
+  display: flex;
+  flex-direction: column;
   width: 100%;
-  table-layout: fixed;
-  border-spacing: 0;
-  border-collapse: collapse;
+  max-width: 1500px;
+  margin-right: auto;
+  margin-left: auto;
 
   &::before {
     position: absolute;
-    top: 0;
-    bottom: 30px;
+    top: 1.25rem;
+    bottom: 1.25rem;
     left: 50%;
     z-index: 0;
     width: 6px;
     margin-left: -4px;
     content: "";
+    @include border-radius(10rem);
     background-color: #ddd;
   }
 
+  /* Row layout and basic grid settings for desktop */
   .timeline-item {
-    display: table-row;
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: 1fr 4rem 1fr;
+    align-items: start;
+    margin-bottom: 1.25rem;
+  }
+
+  /* Wrappers for content */
+  .timeline-date-wrapper {
+    grid-row: 1 / 2;
+    grid-column: 1;
+  }
+
+  .timeline-icon-wrapper {
+    display: flex;
+    grid-column: 2;
+    justify-content: center;
+  }
 
-    &.alt {
+  .timeline-content-wrapper {
+    position: relative;
+    grid-row: 1 / 2;
+    grid-column: 3;
+  }
+
+  /* Alternative settings for reverted row */
+  .timeline-item.right {
+    .timeline-date-wrapper {
+      grid-column: 3;
+      justify-content: left;
+    }
+
+    .timeline-content-wrapper {
+      grid-column: 1;
       text-align: right;
+    }
 
-      &::before {
-        display: none;
-      }
+    .timeline-date {
+      justify-content: left;
+    }
+  }
+
+  /* Content styling */
+  .timeline-date {
+    display: flex;
+    align-items: center;
+    justify-content: right;
+    margin-top: 0.6rem;
+    margin-bottom: 0.6rem;
+
+    i {
+      margin-right: 0.1rem;
+    }
+  }
 
-      .timeline-panel {
-        margin-right: 42px;
-        margin-left: 0;
+  .timeline-date-day {
+    margin-right: 0.4rem;
+  }
+
+  .timeline-icon {
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 2.5rem;
+    height: 2.5rem;
+    line-height: 2.5rem;
+    background-color: #666;
+    border: 2px solid $white;
+    @include border-radius(2.5rem);
+  }
+
+  .timeline-content {
+    padding: 1.25rem;
+    background-color: $white;
+    border: 1px solid #ddd;
+    @include border-radius(5px);
+  }
+
+  @media (max-width: 767px) {
```

Comment:
Please use helpers like `@include media-breakpoint-down(md) {`, etc.

Reply:
@kpodemski Fixed, I don't know why I didn't do it the first time. It's even more precise.